Mathlib/Algebra/Order/GroupWithZero/WithZero.lean

```diff
@@ -93,7 +92,7 @@ lemma WithZero.withZeroUnitsEquiv_strictMono :
 
 /-- Given any linearly ordered commutative group with zero `α`, this is the order isomorphism
 between `WithZero αˣ` with `α`. -/
-@[simps!]
+-- @[simps!]
```

Comment:
What's the motivation for this? Certainly we shouldn't comment it out, either we keep or remove it.

Reply:
Another leftover from some trial, thanks.